Sources/SwiftLanguageService/CodeActions/PackageManifestEdits.swift

```diff
@@ -175,81 +175,22 @@ struct PackageManifestEdits: SyntaxCodeActionProvider {
   ]
 }
 
-fileprivate extension PackageEdit {
+fileprivate extension [SourceEdit] {
   /// Translate package manifest edits into a workspace edit. This can
   /// involve both modifications to the manifest file as well as the creation
   /// of new files.
   /// `snapshot` is the latest snapshot of the `Package.swift` file.
   func asWorkspaceEdit(snapshot: DocumentSnapshot) -> WorkspaceEdit {
```

Comment:
Do we need this any more?

Reply:
Yeah, `CodeAction.edit` expects a `WorkspaceEdit` and all the actions use `asWorkspaceEdit` to convert.

Reply:
How do the existing refactorings work?

Reply:
Same thing they just inline WorkspaceEdit initializer because their argument is trivial.